README.md

```diff
@@ -44,7 +44,15 @@ S3_PREFIX - Mention this different folder from the root
 
 S3_REGION - Default is us-west-1
 
-CRON_TIME - Default is "0 */1 * * *". Runs every hour 0th min.
+CRON_TIME_HOURLY = 0 */1 * * * (every 1 hour)
+CRON_TIME_DAILY = 0 */24 * * * (every 24 hours)
+CRON_TIME_WEEKLY = 0 3 * * SUN (3am on SUNDAY)
+CRON_TIME_MONTHLY = 0 4 1 * * (4am on 1st of every month)
+
+S3_LIFECYCLE_EXPIRATION_DAYS_FOR_HOURLY_BACKUP=1 - retain for 24 hours - 24 copies 
+S3_LIFECYCLE_EXPIRATION_DAYS_FOR_DAILY_BACKUP=7 - retain for 7 days - 7 copies 
+S3_LIFECYCLE_EXPIRATION_DAYS_FOR_WEEKLY_BACKUP=365 - retain 52 copies (1 year)
```

Comment:
@vijayabharathib do we need to store 52 copies. Can we just store last 4 ?

Reply:
yes makes sense. will change it.

Reply:
CHANGED.